projects/npm-tools/packages/npm-scripts/src/utils/createFederationConfig.js

```diff
@@ -220,6 +225,7 @@ module.exports = async function () {
 		plugins: [
 			new ModuleFederationPlugin({
 				exposes: {
+					...prefixExposesPaths(exposes, `./${build.input}/`),
```

Comment:
Do we need the `./` here? I believe it won't change the behavior of `path.relative`.

Reply:
That is for webpack not us :-). We invoke `path.relative` below to make sure we strip any leading `./` (that's `path.relative` convention for its return value) and then we prepend `./` for webpack (see [webpack docs](https://webpack.js.org/concepts/module-federation/#troubleshooting) for more detail; there you can see the leading `./` convention in file paths).